Library/Tests/TestHelpers/TestCase.swift

```diff
@@ -23,6 +23,7 @@ internal class TestCase: FBSnapshotTestCase {
 
   override func setUp() {
     super.setUp()
+//    self.recordMode = true
```

Comment:
just remember to remove this or i'm fine with keeping it permanently for toggling.